save-backend/src/main/resources/application.properties

```diff
@@ -1 +1,18 @@
-server.port = 5000
\ No newline at end of file
+server.port = 5000
+
+spring.h2.console.enabled=true
+
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
Let's move H2 config to `application-dev.properties` under `dev` profile

Reply:
Moved